src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -658,12 +658,14 @@ def merge(self,
             tmp_dir=None,
             additional_args=None,
             merge_control_file=None):
-    # TODO(flowerhack): Integrate some notion of a merge timeout.
     self._push_corpora_from_host_to_target(corpus_directories)
 
     additional_args = copy.copy(additional_args)
     if additional_args is None:
       additional_args = []
+    max_total_time_flag = '%s%d' % (constants.MAX_TOTAL_TIME_FLAG,
```

Comment:
nit: I know we have this pattern of '%s%d' in other places, but we're trying to avoid this now. Let's just do `constants.MAX_TOTAL_TIME_FLAG + str(merge_timeout)` which is more readable. (similarly in other places touched by your PR).